instagram-app/src/components/PostContainer/PostCont.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import CommentSection from "./CommentSection";
+
+
```

Comment:
Another option is to iterate the array of objects and for each instance, pass in a CommentSection component and Post component. This way each post can get an array of comments.